lib/game.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+def initialize
+  @new_board = Board.new
+  @player1 = Player.new
+  @player2 = Player.new
+  @new_board.show_board
+  instructions
+  play
+end
+
+def instructions
```

Comment:
Please see point 1 in my main comment related to separation of UI code from your classes